### DISCUSSION TITLE
How I can have Pair Extraordinaire?

### DISCUSSION THREAD

**Discussion by holedev** [Schweinepriester/github-profile-achievements#651] [Q&A] [answered]
Sr but I don't know what i should do with "Coauthored in a merged pull request". Can somebody help me?

**Comment by tauanesales** [comment, marked as answer]
1. Start a new branch on the project;
2. Make your changes;
3. On your commit message, press enter three times and add `Co-authored-by: username <user@email.com>`

e.g.:
```
git commit -m "Updated README. Added project info.


Co-authored-by: tauanesales <taysales6@gmail.com>"
```
4. Start a Pull Request;
5. Once your PR is merged you should get Pair Extraordinaire.

If this helped you, please add this as an accepted answer.

**Reply by runepiper** [reply]
Didn't work for me

**Reply by holedev** [reply]
> On your commit message, press enter three times and add `Co-authored-by: username <user@email.com>`
> 
> e.g.:
> 
> ```
> git commit -m "Updated README. Added project info.
> 
> 
> Co-authored-by: tauanesales <taysales6@gmail.com>"
> ```
> 
> If this helped you, please add this as an accepted answer.

Oh I did it. Thank you for your comment. Exactly, step by step I did:
- Create a feature "test" in a project that I'm a collaborator
- Change 'readme.md'
- Commit with coauthored as you advise (I think before I don't have **three enter**)
- Finally, push and merge pull request in project

**Reply by tauanesales** [reply]
> Didn't work for me

I updated my answer with more steps. I previously had only explained how to make Coauthored commits, to get the badge you need to have Coauthored commits in a merged Pull Request.

PS.: You can also make coauthored commits through GH interface by just pressing enter three times at the end of your commit message and adding `Co-authored-by: username <user@email.com>` (in the second box, after the commit title).

Hope this helps!
